Makefile

```diff
@@ -179,7 +181,7 @@ smallbuild:
 	@echo "$$SMALLBUILD_HELP_INFO"
 else
 smallbuild: clean
-	hack/make-rules/smallbuild.sh $(WHAT)
+	ENABLE_SMALLBUILD=YES ARCH=$(ARCH) hack/make-rules/crossbuild.sh $(WHAT)
```

Comment:
I still see "hack/make-rules/smallbuild.sh" and "hack/make-rules/crossbuild.sh" in place, and is run with different ways.

Reply:
Forget to remove "hack/make-rules/smallbuild.sh", now smallbuild reuse crossbuild.
You remind me, it is better for smallbuild to reuse build logic rather than crossbuild.